ntoskrnl/mm/pagefile.c

```diff
@@ -330,7 +340,7 @@ MmAllocSwapPage(VOID)
             MiUsedSwapPages++;
             MiFreeSwapPages--;
             KeReleaseGuardedMutex(&MmPageFileCreationLock);
-
+            UpdatePeakCommitment();
```

Comment:
`MmFreeSwapPage` should also call this, to keep the `Total` value accurate

Reply:
yes